backend/spec/models/enrollment_spec.rb

```diff
@@ -160,12 +160,12 @@
             team_members_attributes: [
```

Comment:
The test seems incomplete and do not cover this case : 

- If an administrator wants to modify a family_name, given_name and a demandeur'email, the test will failed. 

Plus it will return an enrollment with only email information for the demandeur and with nil for all others fields. 

https://user-images.githubusercontent.com/43042737/216290027-16d2320b-653b-4850-9d2a-aa0ea8ada1e7.mov

Reply:
that will failed all the time, one solution would be to transfert the owner ship the enrollment to another MCP user with a valid UID. fetch the new user information and update the team member demandeurs information with the new owner information. 
⚠️